diff --git a/ankisyncctl.py b/ankisyncctl.py
index bb98918..51482b3 100755
--- a/ankisyncctl.py
+++ b/ankisyncctl.py
@@ -48,7 +48,8 @@ def passwd(username):
         print("User {} doesn't exist".format(username))
         return
 
-    password = getpass.getpass("Enter password for {}: ".format(username))
+        print("Enter password for {}: ".format(username))
+        password = sys.stdin.readline().rstrip()
     try:
         user_manager.set_password_for_user(username, password)
     except ValueError as error:
diff --git a/ankisyncd/sync_app.py b/ankisyncd/sync_app.py
index 3b07420..86b0e2f 100644
--- a/ankisyncd/sync_app.py
+++ b/ankisyncd/sync_app.py
@@ -752,7 +752,7 @@ def make_app(global_conf, **local_conf):
     return SyncApp(**local_conf)
 
 def main():
-    logging.basicConfig(level=logging.INFO)
+    logging.basicConfig(level=logging.WARNING, stream=sys.stdout)
     from wsgiref.simple_server import make_server
     from ankisyncd.thread import shutdown
     import ankisyncd.config
